libs/wire-api-federation/src/Wire/API/Federation/API/Brig.hs

```diff
@@ -68,7 +68,7 @@ type BrigApi =
     -- (handles can be up to 256 chars currently)
     :<|> FedEndpoint "search-users" SearchRequest SearchResponse
     :<|> FedEndpoint "get-user-clients" GetUserClients (UserMap (Set PubClient))
-    :<|> FedEndpoint "get-mls-clients" MLSClientsRequest (Set ClientId)
+    :<|> FedEndpoint "get-mls-clients" MLSClientsRequest (Set ClientId, Set ClientId)
```

Comment:
This is a breaking federation API change. Should it be versioned, i.e., should it go into a new version 2?

Reply:
Fortunately, we don't yet have a stable federation API version.